Classes/Systems/AppDelegate.swift

```diff
@@ -76,6 +76,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         completionHandler(ShortcutHandler.handle(route: route,
                                                  sessionManager: sessionManager,
                                                  navigationManager: rootNavigationManager))
+
+        ShortcutHandler.configure(application: application, sessionManager: sessionManager)
```

Comment:
Hmm this will only update when you switch via the 3d touch menu, right? It wont update when you're switching users via settings, right?

I actually think we should add the `AppDelegate` as a listener to the session manager, and on `didFocus(...)` call this line to update the shortcuts.

What do you think?

Reply:
Actually it updates when you're switching users via settings.. 
I am okay adding the `AppDelegate` as a listener to the session manager and configuring `ShortcutHandler ` on `didFocus`, but I don't understand what is the problem the actual implementation...

Reply:
It doesn’t, the only way we update the menu item now is in the app delegate which requires a cold start. This PR takes it a step closer and updates when you switch accounts via the shortcut, but if you go to Settings and switch accounts there, the previous account won’t be updated.